packages/blockchain/test/blockValidation.spec.ts

```diff
@@ -124,7 +125,9 @@ describe('[Blockchain]: Block validation tests', () => {
 
   it('should throw if the uncle header is invalid', async () => {
     const consensusDict: ConsensusDict = {}
-    consensusDict[ConsensusAlgorithm.Ethash] = new EthashConsensus(new Ethash())
+    consensusDict[ConsensusAlgorithm.Ethash] = new EthashConsensus(
+      new Ethash() as MinimalEthashInterface,
+    )
```

Comment:
Why do we add this here? Seems out of place for a release doc update

Reply:
The typecheck CI job was failing with an error here, so I fixed it, but more came up.

Reply:
Update: found the root cause for this, there was an Ethash dev dependency version not updated and therefore the corresponding block types in the function signature of `verifyPoW(block: Block,...)` did not match.